R/get_url.R

```diff
@@ -0,0 +1,48 @@
+# get_url_bioconductor_package -------------------------------------------------
+get_url_bioconductor_package <- function(package) {
+
+  get_url_bioconductor_package_2("https://bioconductor.org", package)
+}
+
+# get_url_bioconductor_package_2 -----------------------------------------------
+get_url_bioconductor_package_2 <- function(base_url, package) {
+
+  paste0(base_url, "/packages/release/bioc/html/", package, ".html")
+}
+
+# get_url_cran_package ---------------------------------------------------------
+# CRAN canonical URL
+get_url_cran_package <- function(package) {
```

Comment:
same comment than for the BioConductor URL functions. :-)

Reply:
see above.